store/doc.go

```diff
@@ -0,0 +1,20 @@
+package store
+
+/*
```

Comment:
This package godoc is incorrect. I'd remove it entirely.

Reply:
I agree that formatting is incorrect (@mslipper doc comments must precede the `package` statement, and preferably one should use the `//` comment syntax), but why removing the docs altogether? I'd suggest to improve it instead :)

Reply:
1. The package name is wrong, it's not `storeutil`.
2. It's in the wrong location. The package godoc comes _before_ the package name.
3. The actual godoc is describing the addition of the utility functions added. The actual function is far more complicated and feature-rich.